tests/dummy/app/templates/button.hbs

```diff
@@ -63,4 +65,94 @@
     \{{paper-button raised=true label="Blockless version"}}
   &lt;/p&gt;{{/code-block}}
 </div>
+
+<h2>Usage</h2>
+
+<table>
+  <thead>
+    <tr>
+      <th>API</th>
+      <th>Type</th>
+      <th>Description</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td><strong>accent</strong></td>
+      <td>boolean</td>
+      <td>Display in the theme's accent color</td>
+    </tr>
+    <tr>
+      <td><strong>bubbles</strong></td>
+      <td>boolean</td>
+      <td>Determines whether the Ember click event handler bubbles. Default is {{#code-inline}}undefined{{/code-inline}}, which bubbles</td>
+    </tr>
+    <tr>
+      <td><strong>disabled</strong></td>
+      <td>boolean</td>
+      <td>Whether the button is displayed as disabled and does not accept clicks</td>
+    </tr>
+    <tr>
+      <td><strong>fab</strong></td>
+      <td>boolean</td>
+      <td>Display as a Floating Action Button</td>
+    </tr>
+    <tr>
+      <td><strong>href</strong></td>
+      <td>string</td>
+      <td>Displays the button as an {{#code-inline}}&lt;a&gt;{{/code-inline}} link to the specified destination URL</td>
+    </tr>
+    <tr>
+      <td><strong>iconButton</strong></td>
+      <td>boolean</td>
+      <td>Set when the contents contains an icon and adjusts CSS appropriately</td>
+    </tr>
+    <tr>
+      <td><strong>label</strong></td>
+      <td>string</td>
+      <td>Set the content of the button when used as a blockless component</td>
+    </tr>
+    <tr>
+      <td><strong>mini</strong></td>
+      <td>boolean</td>
+      <td>Display as a mini-sized button. Implies {{#code-inline}}fab{{/code-inline}}, unless {{#code-inline}}fab{{/code-inline}} is explicity set falsy.</td>
+    </tr>
+    <tr>
+      <td><strong>noInk</strong></td>
+      <td>boolean</td>
+      <td>Suppresses the ripple effect when clicked</td>
+    </tr>
+    <tr>
+      <td><strong>onClick</strong></td>
+      <td>closure</td>
+      <td>Action sent when the button is clicked</td>
+    </tr>
+    <tr>
+      <td><strong>primary</strong></td>
+      <td>boolean</td>
+      <td>Display as the primary button, more prominent that other buttons</td>
+    </tr>
+    <tr>
+      <td><strong>raised</strong></td>
+      <td>boolean</td>
+      <td>Display button with a 3-D effect</td>
+    </tr>
+    <tr>
+      <td><strong>target</strong></td>
+      <td>string</td>
+      <td>Sets the {{#code-inline}}&lt;a&gt;{{/code-inline}} link target attribute, such as {{#code-inline}}"_blank"{{/code-inline}}</td>
+    </tr>
+    <tr>
+      <td><strong>type</strong></td>
+      <td>string</td>
+      <td>Sets the html5 {{#code-inline}}type{{/code-inline}} attribute.</td>
+    </tr>
+    <tr>
+      <td><strong>warn</strong></td>
+      <td>boolean</td>
+      <td>Display in the theme's warn color</td>
```

Comment:
accent, primary and warn are part of a mixin and are present in many other components. We may need to find a better way to show common attributes to avoid repeating ourselves over and over, but for now listing them here seems the best thing to do.